### DISCUSSION TITLE
Height of resize line on DataTable

### DISCUSSION THREAD

**Discussion by hennievw** [primefaces/community#2621] [PrimeReact] [answered]
When you provide a header area for a DataTable and you resize a column, the line that it renders to indicate the size overlaps the header area. Is there any way to only limit it to the actual column/rows area?

![gridresize](https://github.com/user-attachments/assets/0e40fa68-cb97-46a4-8b96-eb0fc1d925f6)

**Comment by qwadrox** [comment, marked as answer]
@hennievw 
You can write some ugly javascript code maybe: 

https://stackblitz.com/edit/brwhrx?file=src%2FApp.jsx

```ts
const resizeResizer = (event) => {
    const innerTableHeight = document.getElementsByClassName(
      'p-datatable-wrapper'
    )[0]?.offsetHeight;

    if (innerTableHeight) {
      // Create a new style element
      const style = document.createElement('style');
      style.type = 'text/css';

      // Define the new CSS rule with the additional height
      const newHeightRule = `
            .p-column-resizer-helper {
                height: ${innerTableHeight}px !important;
                bottom: 0 !important;
                top: unset !important;
            }
        `;

      // Append the rule to the style element
      style.appendChild(document.createTextNode(newHeightRule));

      // Append the style element to the head of the document
      document.head.appendChild(style);
    }
  };
```

Or if you have a defined number of rows in advance, maybe you can play with the CSS: 
```css
.mytable > [class~='p-column-resizer-helper'] {
  height: 90% !important;
  bottom: 0 !important;
  top: unset !important;
}
```

This is whats happening internally: [const onColumnResize = (event) => {](https://github.com/primefaces/primereact/blob/26240b5bbc703daccc3431682b49ccd4a1f0fac2/components/lib/datatable/DataTable.js#L525-L536)
It sets the height of the resizer based on the primary div of the table.

**Reply by hennievw** [reply]
Thanks for the response
